modules/ravestate_verbaliser/__init__.py

```diff
@@ -0,0 +1,20 @@
+from os.path import dirname, realpath, join
+
+from ravestate import registry
+from ravestate.state import state
+from ravestate_verbaliser import verbaliser
+
+
+# TODO this is just a testdummy for a verbaliser state
```

Comment:
Please empty this file. Since `ravestate_verbalizer` is a utility module, it doesn't make sense for it to register any states, right?

Reply:
Although creating some semantic output state like `verbalizer:intent` is certainly tempting 😁